tests/manager/manager-appname.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+MongoDB\Driver\Manager - appname
+--FILE--
+<?php
+$name1 = str_repeat( "PHONGO", 20 );
+$name2 = str_repeat( "PHONGO", 20 );
+$manager = new \MongoDB\Driver\Manager("mongodb://localhost:27017/?appname={$name1}", [ 'appname' => "2-{$name2}" ] );
```

Comment:
Since this test is actually connecting, we should be using STANDALONE here with the usual boilerplate SKIPIF and includes.